spec/playlist_spec.rb

```diff
@@ -194,4 +194,20 @@
     expect(playlist.master?).to be true
     expect(playlist.items.size).to eq 6
   end
+
+  it 'should return the total duration of a playlist' do
+    playlist = M3u8::Playlist.new
+
+
+    item = M3u8::SegmentItem.new({ duration: 10.991, segment: 'test_01.ts' })
+    playlist.items.push item
+    item = M3u8::SegmentItem.new({ duration: 9.891, segment: 'test_02.ts' })
```

Comment:
Redundant curly braces around a hash parameter.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.